server/src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -1234,7 +1236,15 @@ Quaternion extendedKneeYawRoll(Quaternion knee, Quaternion ankle) {
 	 * @param hip the third Quaternion
 	 * @return the rotated Quaternion
 	 */
-	Quaternion extendedPelvisYawRoll(Quaternion leftKnee, Quaternion rightKnee, Quaternion hip) {
+	private Quaternion extendedPelvisYawRoll(
+		Quaternion leftKnee,
+		Quaternion rightKnee,
+		Quaternion hip
+	) {
+		leftKnee = leftKnee.clone();
+		rightKnee = rightKnee.clone();
+		hip = hip.clone();
```

Comment:
Sorta same as last comment, but it shouldn't be necessary to clone all three, only hip since it's the only one being output.

Reply:
I'm modifying the other twos at line 1254 and 1257